files/brews/zlib.rb

```diff
@@ -3,7 +3,7 @@
 class Zlibphp < Formula
   homepage 'http://www.zlib.net/'
   url 'http://zlib.net/zlib-1.2.8.tar.gz'
```

Comment:
'fossils' 😂
Thanks @convenient I'll apply that patch to this PR and credit you.